### ISSUE TITLE
centerMode true, infinite false : current-slide does not have active-slide class, also has aria-hidden="true"

### ISSUE THREAD

**Issue by kakt00c** [kenwheeler/slick#2530]
When settings has centerMode true, infinite false, current-slide does not have active-slide class, also has aria-hidden="true"

http://jsfiddle.net/a39y7zx4/

====================================================================

#### Steps to reproduce the problem

1. init carousel with settings: 
{
    infinite: false,
    centerMode: true,
    accessibility: true
}
2. Inpect DOM

====================================================================

#### What is the expected behaviour?

Current slide should have active-slide class, also aria-hidden should be false, so user can tab throw the slide

====================================================================

#### What is observed behaviour?
Current slide does not have active-slide class, also aria-hidden set to true, so user can not tab throw the slide

**Comment by kakt00c** [comment]
I have reviewed code logic, and find out the problem place. 
Please take a look at my comment on the piece below. 


    if (_.options.centerMode === true) {
        // ...
        if (_.options.infinite === true) {
            if (index >= centerOffset && index <= (_.slideCount - 1) - centerOffset) {
                _.$slides
                    .slice(index - centerOffset, index + centerOffset + 1)
                    .addClass('slick-active')
                    .attr('aria-hidden', 'false');
            } else {
                indexOffset = _.options.slidesToShow + index;
                allSlides
                    .slice(indexOffset - centerOffset + 1, indexOffset + centerOffset + 2)
                    .addClass('slick-active')
                    .attr('aria-hidden', 'false');

            }
            // ...
        }

        // and we do not have ELSE for the infinite mode!
        // so if we have center mode true, infinite mode false,
        // than we do not have slide with class slick-active, and every slide has aria-hidden=false !

        // please help me to resolve this.


        // ...
    } else {
        // ...
    }

**Comment by leggomuhgreggo** [comment]
Cool thanks for the debugging @kakt00c

**Comment by akrawchyk** [comment]
Related https://github.com/kenwheeler/slick/issues/2812
